lib/TokenManager.ts

```diff
@@ -153,7 +154,7 @@ function validateToken(token: Token) {
   if (!isObject(token) ||
       !token.scopes ||
       (!token.expiresAt && token.expiresAt !== 0) ||
-      (!isIDToken(token) && !isAccessToken(token))) {
+      (!isIDToken(token) && !isAccessToken(token) && !isRefreshToken(token))) {
     throw new AuthSdkError('Token must be an Object with scopes, expiresAt, and an idToken or accessToken properties');
```

Comment:
nit: Should the error say "and idToken, accessToken or refreshToken properties"?